extensions/sql-database-projects/src/models/project.ts

```diff
@@ -122,6 +122,11 @@ export class Project implements ISqlProject {
 		// check if this is a new msbuild sdk style project
 		this._isMsbuildSdkStyleProject = this.CheckForMsbuildSdkStyleProject();
 
+		await this.loadFilesInProject();
```

Comment:
think there's any value in keeping a consistent model for loading values?  right now, it's both doing `this._something = loadSomething()` and `loadSomethingElse()` where "loadSomethingElse()" does the assignment within.

I think I like the idea of the former model.

Reply:
sure we can go with that model since that's what we already have for sqlcmd variables. I was gonna need to update loadFilesInProject to return the files anyway to reuse it for excluding files.